test_util/run-all

```diff
@@ -113,7 +114,7 @@ RET=$?
 mkdir -p $WORKDIR/test/vagrant-logs
 if [ $RET -ne 0 ]; then
   echo "TEST FAILED"
-  for i in boot m1 m2 m3 a1 a2; do
+  for i in boot m1 m2 m3 a1 a2 p1; do
```

Comment:
note: Will need to update the CI log collector as well that outputs to the actual TeamCity log in chunks